packages/lending/src/protocol.portfolio.ts

```diff
@@ -135,6 +163,18 @@ export class Borrow implements AssetInfo {
     return formatPercentage(this.apy);
   }
 
+  get formattedLstAPY() {
+    return formatPercentage(this.lstApy);
+  }
+
+  get grossApy() {
+    return this.grossApys[0];
```

Comment:
What's the meaning of `grossApys[0]` and why do we get this value only ?

Reply:
There are two types of APY for borrowing: `variableBorrowAPY` and `stableBorrowAPY` in aave-like protocol, so the grossApy is calculated in the same manner.
but the trush is that stable APY is almost unsupported, so we only use first one `variableBorrowAPY`